.gitignore

```
@@ -17,4 +17,4 @@ dist
 ## Terraform ##
 .terraform
 .terraform.lock.hcl
-.tfvars
+*.tfvars
```

A bit of scope creep but this would fully align us with the [`.gitignore` file in cisagov/skeleton-tf-module](https://github.com/cisagov/skeleton-tf-module/blob/develop/.gitignore).

```suggestion
terraform.tfstate
terraform.tfstate.backup
*.tfvars
```